frontend/src/pages/projects/types.ts

```diff
@@ -64,8 +65,8 @@ export type StorageData = {
   mountPath?: string;
   existingName?: string;
   existingPvc?: PersistentVolumeClaimKind;
+  accessMode?: AccessMode;
```

Comment:
this will be a conflict with the previous pr, unfortunately. (it lists it as a string)